packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/emitting/NotificationsEmitter.java

```diff
@@ -47,6 +49,10 @@ public void onDestroy() {
     mNotificationManager.removeListener(this);
   }
 
+  public void getLastNotificationResponseAsync(Promise promise) {
```

Comment:
should this be an `@ExpoMethod`?

Reply:
Of coourse! Thank you for catching that, I was so worked up on iOS not working I forgot to test Android!